### ISSUE TITLE
Is there a place to try this out?

### ISSUE THREAD

**Issue by ardent-android** [imdevan/hola#2]
Seems like a really cool project. Do you guys have a link... or the steps to running the application? I tried running it, but it doesn't send any requests to Twilio, and for some reason is gives a variable io not found reference error, i take it that socket io is not configured correctly?

**Comment by Vatyx** [comment]
That's interesting. We host it on cloud9 and run it. I can push that and
you can pull that I can let you know when that happens.

**Comment by ardent-android** [comment]
<img width="702" alt="screen shot 2015-10-21 at 7 13 39 pm" src="https://cloud.githubusercontent.com/assets/13618482/10655257/ead9c738-7827-11e5-88a6-15c4929be48e.png">

Just curious is that supposed to happen? Also do you guys get a ReferenceError: Can't find variable: io error?

Also want to know... If I wanted to run this app I would just npm install then gulp build then gulp server. Right?

**Comment by Vatyx** [comment]
That happens because we didn't set a case for when there is nothing in the
text field. It sends an empty string to the translator and that's what it
receives. And once again I think it's because the version we used is
updated on Cloud9 and not on Github.

**Comment by ardent-android** [comment]
In Cloud9, what commands did you use to run your app?

**Comment by ardent-android** [comment]
Just wondering if you guys could post the updated code to this github repo

**Comment by Vatyx** [comment]
Are you planning on using this for your own project?

**Comment by ardent-android** [comment]
No, im not working on any projects right now. Just wanted to mess around with twilio apps (like cosmos browser).

**Comment by ardent-android** [comment]
Is this a twiml app?

**Comment by imdevan** [comment]
Hi Ardent! Sorry for not getting back to you sooner. I'm actually not sure if this is a twiml app, I know we process the request via our own node server if that helps answer you're question at all!
